src/morello-issues/README.md

```diff
@@ -21,6 +21,26 @@ CheriBSD on Morello:
   it's generally best to start over as performance is awful once timeouts
   start.
 
+- The HDMI transmitter on the board might fail to detect if a monitor is
+  plugged in.
+  In such a case, the monitor will turn blank when the CheriBSD kernel starts
+  printing messages to the console, after the CheriBSD boot loader menu.
+  A user can configure the kernel to ignore hot-plugging detection and assume
+  that the attached monitor can receive HDMI input.
+  To do that, add the following line to `/boot/loader.conf` and reboot your
+  board:
+  ```
+  hw.tda19988.broken_hpd="1"
+  ```
+
+- The HDMI transmitter on the board might fail to detect monitor's resolution.
```

Comment:
```suggestion
- The HDMI transmitter on the board might fail to detect the monitor's resolution.
```
Probably this is in the case that HPD is also broken; perhaps worth a cross-reference if so?